index.js

```diff
@@ -152,13 +156,32 @@ function createMainWindow() {
 	return win;
 }
 
-app.on("browser-window-created", (event, win) => {
+app.once("browser-window-created", (event, win) => {
 	loadPlugins(win);
 
-	win.webContents.on("did-fail-load", () => {
+	win.webContents.on("did-fail-load", (
+		event,
+		errorCode,
+		errorDescription,
+		validatedURL,
+		isMainFrame,
+		frameProcessId,
+		frameRoutingId,
+	) => {
+		const log = {
+			error: "did-fail-load",
+			event,
+			errorCode,
+			errorDescription,
+			validatedURL,
+			isMainFrame,
+			frameProcessId,
+			frameRoutingId,
+		};
 		if (is.dev()) {
-			console.log("did fail load");
+			console.log(log.toString());
```

Comment:
Because `log` is an object, this will just log `[object Object]`, any reason not to call `JSON.stringify` directly? I also got `Error: Failed to serialize arguments` on line 184 so it might be better to pass it already stringified (it will end up serialized anyway!)

Reply:
This actually a big oversight on my part, sorry about that..